src/components/EventList/components/EventItem/index.jsx

```diff
@@ -16,6 +16,7 @@ const EventItem = ({ event }) => {
       onClick={() => setExpanded((e) => !e)}
       className={styles.event_item}
       tabIndex={0}
+      onKeyDown={(e) => (e.key === 'Enter' ? setExpanded((e) => !e) : {})}
```

Comment:
You can check if the `target` of the event is the same as the `currentTarget`, to see whether this event was triggered on this element or if it bubbled up from a child element. More info: https://developer.mozilla.org/en-US/docs/Web/API/Event/currentTarget
```suggestion
      onKeyDown={(e) => (e.key === 'Enter' && e.currentTarget === e.target ? setExpanded((e) => !e) : {})}
```

Also, I think this anonymous function would be easier to read if we use a normal function body:
```suggestion
      onKeyDown={(e) => {
          if (e.key === 'Enter') {
              setExpanded((e) => !e);
          }
      }}
```